src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -139,7 +148,7 @@ protected function getCellXfsSectionContent()
         $content = '<cellXfs count="' . count($registeredStyles) . '">';
 
         foreach ($registeredStyles as $style) {
-            $content .= '<xf numFmtId="0" fontId="' . $style->getId() . '" fillId="0" borderId="0" xfId="0"';
+            $content .= '<xf numFmtId="0" fontId="' . $style->getId() . '" fillId="' . $style->getId() .  '" borderId="0" xfId="0"';
```

Comment:
Well, I guess the specs tell you that you can/should reuse common parts. Whether it's fills, borders, shared strings, it seems like this is what OOXML is recommending (and that makes sense).